engine/src/main.rs

```diff
@@ -166,6 +166,22 @@ async fn run_main(settings: Settings) -> anyhow::Result<()> {
 					expected_eth_chain_id,
 				)?
 			};
+			let arb_client = {
+				let expected_arb_chain_id = web3::types::U256::from(
+					state_chain_client
+						.storage_value::<pallet_cf_environment::ArbitrumChainId<state_chain_runtime::Runtime>>(
+							state_chain_client.latest_finalized_block().hash,
+						)
+						.await
+						.expect(STATE_CHAIN_CONNECTION),
+				);
+				EthRetryRpcClient::<EthRpcSigningClient>::new(
```

Comment:
Just recalled this from when you were debugging the other day. This should be renamed to Evm, and they log message should be generic as currently they are `eth_rpc` and `eth_subscribe`

Reply:
Created an issue for this (PRO-1221). Will do this in another PR